libtenzir/builtins/operators/slice.cpp

```diff
@@ -376,10 +378,39 @@ class plugin final : public virtual operator_plugin<slice_operator> {
     stride = parse_int(true);
     return std::make_unique<slice_operator>(begin, end, stride);
   }
+
+  auto
+  make(invocation inv, session ctx) const -> failure_or<operator_ptr> override {
+    auto begin = std::optional<int64_t>{};
+    auto end = std::optional<int64_t>{};
+    auto stride = std::optional<int64_t>{};
+    // TODO: Range selector syntax
+    TRY(argument_parser2::operator_(name())
+          .add(begin, "int")
+          .add(end, "int")
+          .add(stride, "int")
+          .parse(inv, ctx));
+    return std::make_unique<slice_operator>(begin, end, stride);
+  }
+};
+
+struct reverse_plugin final : public virtual operator_factory_plugin {
+  auto name() const -> std::string override {
+    // FIXME: `operator_parser_plugin` does not play well with duplicate
+    // names and `tql2.` prefix
```

Comment:
I assume this is a problem because there's a `<string>.reverse()` method?

Reply:
Yes, me and @jachris decided on the huddle, to go with `sliced` and `reversed` for the tql2 operators till the issue is properly fixed in the plugin manager.

Reply:
This is now fixed with @jachris' recent PR that allows for setting the name separately.